### PR TITLE
Fix solve_undistorted_squared_radius stability

### PR DESCRIPTION
**Pull request by nitrotm**
See issue #428 for details.

### CODE REVIEW

**Comment by simonfuhrmann** [comment]
Thank you for contributing this. @flanggut, can you review these changes?

File: libs/sfm/bundler_common.cc [hunk 1]
```diff
@@ -281,27 +281,32 @@ namespace {
     double solve_undistorted_squared_radius (double const r2,
         double const k1, double const k2)
     {
-        // We use bisection as we can easily find a pretty good initial guess,
-        // and we don't want to compute all roots for a 5th degree polynomial.
-
         // Guess initial interval upper and lower bound
         double lbound = r2, ubound = r2;
-        while (distort_squared_radius(lbound, k1, k2) > r2)
+        while (distort_squared_radius(lbound, k1, k2) > r2) {
```
Comment (flanggut) [inline libs/sfm/bundler_common.cc]: MVE has opening and closing braces on a separate line.

```
while ( )
{
}
```

File: libs/sfm/bundler_common.cc [hunk 2]
```diff
@@ -281,27 +281,32 @@ namespace {
...
+            ubound = lbound;
             lbound /= 1.05;
-        while (distort_squared_radius(ubound, k1, k2) < r2)
+        }
+        while (distort_squared_radius(ubound, k1, k2) < r2) {
```
Comment (flanggut) [inline libs/sfm/bundler_common.cc]: same as above

**Comment by nitrotm** [comment]
Sure, I updated the code.

**Comment by flanggut** [comment]
Thanks!

**Pull request merged**